libdebug/state/thread_context.py

```diff
@@ -91,6 +89,38 @@ def process_id(self: ThreadContext) -> int:
         """The process ID of the thread."""
         return self.context.process_id
 
+    @property
+    def pid(self: ThreadContext) -> int:
+        """The process ID of the thread."""
+        return self.context.process_id
+
+    @property
+    def thread_id(self: ThreadContext) -> int:
+        """The thread ID."""
+        return self._thread_id
+
+    @property
+    def tid(self: ThreadContext) -> int:
+        """The thread ID."""
+        return self._thread_id
+
+    @property
+    def signal(self: ThreadContext) -> str | None:
+        """The signal will be forwarded to the thread."""
+        return None if self._signal_number == 0 else resolve_signal_name(self._signal_number)
+
+    @signal.setter
+    def signal(self: ThreadContext, signal: str | int) -> None:
+        """Set the signal to forward to the thread."""
+        if self._signal_number != 0:
+            liblog.debugger(
+                f"Overwriting signal {resolve_signal_name(self._signal_number)} with {resolve_signal_name(signal) if isinstance(signal, int) else signal}."
+            )
+        if isinstance(signal, str):
+            signal = resolve_signal_number(signal)
+        self._signal_number = signal
+        self.context._resume_context.threads_with_signals_to_forward.append(self.process_id)
```

Comment:
process_id?
Shouldn't it be thread_id?

Reply:
Can we get tests for this feature?

Reply:
I am actually working on that. I will add tests as soon as possible. I completely forgot them, ops.

Reply:
cdb3d77cbd98acc7d275c2a40883ff4e4368d06e